routes/api.php

```diff
@@ -116,6 +116,12 @@
     // Route::patch('/services/{uuid}', [ServicesController::class, 'update_by_uuid'])->middleware([IgnoreReadOnlyApiToken::class]);
     Route::delete('/services/{uuid}', [ServicesController::class, 'delete_by_uuid'])->middleware([IgnoreReadOnlyApiToken::class]);
 
+    Route::get('/services/{uuid}/envs', [ServicesController::class, 'envs']);
+    Route::post('/services/{uuid}/envs', [ServicesController::class, 'create_env'])->middleware([IgnoreReadOnlyApiToken::class]);
+    Route::patch('/services/{uuid}/envs/bulk', [ServicesController::class, 'create_bulk_envs'])->middleware([IgnoreReadOnlyApiToken::class]);
+    Route::patch('/services/{uuid}/envs', [ServicesController::class, 'update_env_by_uuid']);
```

Comment:
Shouldn't this also use `->middleware([IgnoreReadOnlyApiToken::class]);` ? It's not the case on `/applications/{uuid}/envs` but I'm pretty sure both should have that middleware

Reply:
true, it's a mutation so it should definitely be in there. thanks!